src/cuga/backend/llm/models.py

```
@@ -461,6 +461,49 @@ def _get_base_url(self, model_settings: Dict[str, Any], platform: str) -> str:
+        # Global env var override
+        env_bundle = os.environ.get("CUGA_SSL_CA_BUNDLE", "").strip()
+        if env_bundle:
+            return env_bundle
+
+        # Global TOML setting via dynaconf
+        try:
+            toml_bundle = settings.connections.ssl_ca_bundle
+            if toml_bundle and str(toml_bundle).strip():
+                return str(toml_bundle).strip()
+        except Exception:
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)